EventKit/eventkit-cloud
# updated recipe and images

# Description of Improvement

Uses newer version of gdalutils which fixes clipping raster issue.

## How to test

Create an export with a very small raster dataset that fails on master, it should pass fine from this branch. 

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
